File: ruby-sinatra/api/webhooks_api.rb
require 'json'


MyApp.add_route('POST', '/api/v1/zulip-outgoing-webhook', {
  "resourcePath" => "/Webhooks",
  "summary" => "Outgoing webhooks",
  "nickname" => "zulip_outgoing_webhooks", 
  "responseClass" => "inline_response_200",
  "endpoint" => "/zulip-outgoing-webhook", 
  "notes" => "Outgoing webhooks allows to build or set up Zulip integrations which are notified when certain types of messages are sent in Zulip. ",
  "parameters" => [
    ]}) do
  cross_origin
  # the guts live here

  {"message" => "yes, it worked"}.to_json
end

File: scala-play-server/app/api/WebhooksApi.scala
package api

import play.api.libs.json._
import model.InlineResponse200

@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"))
trait WebhooksApi {
  /**
    * Outgoing webhooks
    * Outgoing webhooks allows to build or set up Zulip integrations which are notified when certain types of messages are sent in Zulip. 
    */
  def zulipOutgoingWebhooks(): InlineResponse200
}
